src/Logging.jl

```diff
@@ -123,7 +123,7 @@ write_log(syslog::SysLog, color::Symbol, msg::AbstractString) = send(syslog.sock
 write_log{T<:IO}(output::T, color::Symbol, msg::AbstractString) = (println(output, msg); flush(output))
 write_log(output::Base.TTY, color::Symbol, msg::AbstractString) = (Base.println_with_color(color, output, msg); flush(output))
 
-function formatlog(format::AbstractString, logging_fields::Dict)
+function formatlog(format::AbstractString, logging_fields::Dict{AbstractString,ANY})
```

Comment:
`Dict{AbstractString,Any}` is good, I'm pretty sure every type will convert to a string